### PR TITLE
DVCSMP-2490 Update generic Z-Wave sensor DTHs

### PR DESCRIPTION
**Pull request by mckeed**
Adds support for NotificationReports to Z-Wave Sensor and more encapsulations to other generic sensor DTHs

### CODE REVIEW

File: devicetypes/smartthings/zwave-motion-sensor.src/zwave-motion-sensor.groovy [hunk 1]
```diff
@@ -178,6 +182,43 @@ def zwaveEvent(physicalgraph.zwave.commands.sensormultilevelv5.SensorMultilevelR
 	createEvent(map)
 }
 
+def zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityMessageEncapsulation cmd) {
+	def encapsulatedCommand = cmd.encapsulatedCommand(commandClassVersions)
+	// log.debug "encapsulated: $encapsulatedCommand"
+	if (encapsulatedCommand) {
+		state.sec = 1
```
Comment (dkirker) [inline devicetypes/smartthings/zwave-motion-sensor.src/zwave-motion-sensor.groovy:189]: I noted in the other review, would state.sec = true be more consistent?

File: devicetypes/smartthings/zwave-sensor.src/zwave-sensor.groovy [hunk 2]
```diff
@@ -198,23 +308,44 @@ def zwaveEvent(physicalgraph.zwave.commands.batteryv1.BatteryReport cmd) {
 	createEvent(map)
 }
 
+def zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityMessageEncapsulation cmd) {
+	def encapsulatedCommand = cmd.encapsulatedCommand(commandClassVersions)
+	if (encapsulatedCommand) {
+		state.sec = 1
```
Comment (dkirker) [inline devicetypes/smartthings/zwave-sensor.src/zwave-sensor.groovy:314]: Same as above.

**Review by dkirker — CHANGES_REQUESTED** [review]
Really just some minor things. I guess this overlaps with https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/2315 so that would be the one that you would use?

**Comment by mckeed** [comment]
I've moved these changes to #2315

**Pull request closed without merging**